internal/notification/channels/twilio/channel.go

```diff
@@ -2,6 +2,8 @@ package twilio
 
 import (
 	"github.com/kevinburke/twilio-go"
+	newTwilio "github.com/twilio/twilio-go"
```

Comment:
Note: We don't actually need to pull in `twilio-go`, the existing kevinburke lib supports the verify API. The twilio-go lib is nice and has better support, but avoiding a new dep is probably the right move.

Reply:
We might just move to the twilio-go lib. Probably at the time we integrated twilio, the lib either did not exist yet or was in an early stage